ivy/array/sorting.py

```diff
@@ -16,8 +16,42 @@ def argsort(
         *,
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
-        return ivy.argsort(self._data, axis, descending, stable, out=out)
+        """
+        ivy.Array instance method variant of ivy.argsort. This method simply wraps the
+        function, and so the docstring for ivy.argsort also applies to this method
+        with minimal changes.
+        
+        Parameters
+        ----------
+        self 
+            input array.
+        axis
+            axis along which to sort. If set to ``-1``, the function must sort along the last axis. Default: ``-1``.
```

Comment:
this line is too long, please update it so that one line only has up to 88 characters, rest of the content would go to a new line.